Purchases/Networking/SubscribersAPI.swift

```diff
@@ -80,9 +86,17 @@ class SubscribersAPI {
               observerMode: Bool,
               subscriberAttributes subscriberAttributesByKey: SubscriberAttributeDict?,
               completion: @escaping BackendCustomerInfoResponseHandler) {
+        let attributionStatus = self.attributionFetcher.authorizationStatus
+        var subscriberAttributesByKey = subscriberAttributesByKey ?? [:]
+        let consentStatus = SubscriberAttribute(withKey: ReservedSubscriberAttribute.consentStatus.rawValue,
+                                                value: attributionStatus.description,
+                                                dateProvider: self.dateProvider)
+        subscriberAttributesByKey[ReservedSubscriberAttribute.consentStatus.rawValue] = consentStatus
```

Comment:
nitpick: we have a [var called key](https://github.com/RevenueCat/purchases-ios/blob/main/Purchases/SubscriberAttributes/ReservedSubscriberAttributes.swift#L20) that's an alias for `rawValue` for readability

```suggestion
        subscriberAttributesByKey[ReservedSubscriberAttribute.consentStatus.key] = consentStatus
```

Reply:
✅